### ISSUE TITLE
committer is crashing in a new way

### ISSUE THREAD

**Issue by lukemarsden** [dotmesh-io/dotscience-committer#7]
```
Traceback (most recent call last):
  File "./hack.py", line 471, in <module>
    main()
  File "./hack.py", line 465, in main
    controllerNotebook()
  File "./hack.py", line 450, in controllerNotebook
    commitNotebook(notebook)
  File "./hack.py", line 375, in commitNotebook
    commit(values, metadata, notebook)
  File "./hack.py", line 251, in commit
    dataDotMetadata["type"] = "dotscience.%s" % (kind,)
NameError: name 'kind' is not defined
```

**Comment by lukemarsden** [comment]
ah looks like this was fixed in 4604cf8b2248fabfb20028144ee2b6cfbbb6574a

**Comment by lukemarsden** [comment]
bumped agent https://github.com/dotmesh-io/dotscience-agent/commit/3791ec86ae5445e7a7fbac03dc6143fe2f4939ec
